### ISSUE TITLE
Laravel 7 release 

### ISSUE THREAD

**Issue by garrettgjb** [nunomaduro/phpinsights#375]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| Library version? | master

This library has not been released since updating to laravel 7. Can someone release it?

**Comment by nunomaduro** [comment]
I gonna to double check this tomorrow.

@szepeviktor @olivernybroe @Jibbarth sorry to ping and for being lazy in not check open issues. But there is anything blocking here? If not, I can do the release tomorrow.

**Comment by szepeviktor** [comment]
I have no clue :) Go on!

**Comment by Jibbarth** [comment]
@nunomaduro check https://github.com/nunomaduro/phpinsights/issues/368#issuecomment-594808048 

I tested installation on a fresh install of laravel but we still have conflict 😥 I don't know how we can fixe them directly

**Comment by owenvoke** [comment]
@Jibbarth, what version of PHP are you installing with?

**Comment by Jibbarth** [comment]
@owenvoke I tested on php73 and php74. Maybe I'm wrong, I'll retry 😅

**Comment by Jibbarth** [comment]
Yup, I still have issue :slightly_frowning_face: @owenvoke You didn't reproduce it ?

**Comment by owenvoke** [comment]
Currently on a train, but I'll check shortly. 👍

**Comment by olivernybroe** [comment]
I just tested on php7.4 and it also gives dependency problems for me when using a fresh laravel installation.

**Comment by owenvoke** [comment]
Looks like it might be an issue with PHPUnit? PHPUnit 8 requires `sebastian/version ^2.0.1`, whereas PHPUnit 9 uses `sebastian/version ^3.0` 🤔 

By default, Laravel apps require `^8.5.2`, so PHPUnit 9 won't install. By updating Laravel's `composer.json` to use `^8.5.2|^9.0`, PHP Insights is then installable. But not sure that's really right as a workaround. 😬

**Comment by owenvoke** [comment]
Also, a heads up, `composer/composer 1.10.0` just released, so `minimum-stability` can be set back to stable instead of RC. 👍

**Comment by nunomaduro** [comment]
@owenvoke Gonna work on the release right now!

**Comment by nunomaduro** [comment]
The version `v1.14.0` is out.

**Comment by mbalandis** [comment]
@nunomaduro 

```
    - Installation request for symfony/console (locked at v5.0.5) -> satisfiable by symfony/console[v5.0.5].
    - Installation request for nunomaduro/phpinsights ^1.14 -> satisfiable by nunomaduro/phpinsights[v1.14.0].
    - Conclusion: don't install phploc/phploc 5.0.0
    - Can only install one of: sebastian/version[3.0.0, 2.0.1].
    - Can only install one of: sebastian/version[3.0.0, 2.0.1].
    - Can only install one of: sebastian/version[3.0.0, 2.0.1].
    - Conclusion: install sebastian/version 3.0.0|install phploc/phploc 5.0.0
    - Installation request for sebastian/version (locked at 2.0.1) -> satisfiable by sebastian/version[2.0.1].
```

still as of v1.14.0. I cannot install it on Laravel 7. But the error is much much shorter now. Any ideas?

**Comment by owenvoke** [comment]
@mbalandis, are you using PHPUnit 9? 🙂

**Comment by Jibbarth** [comment]
@mbalandis Please look here: https://phpinsights.com/get-started.html#within-laravel

> Note for Laravel 7 users : phpinsights requires PHP 7.3+, and PHPUnit 9.0+

You need to first upgrade phpunit to v9

**Comment by mbalandis** [comment]
@owenvoke @Jibbarth 
phpunit 9 doesn't install as well. 

```
  Problem 1
    - Installation request for sebastian/version (locked at 2.0.1) -> satisfiable by sebastian/version[2.0.1].
    - phpunit/phpunit 9.0.0 requires sebastian/version ^3.0 -> satisfiable by sebastian/version[3.0.0].
    - phpunit/phpunit 9.0.1 requires sebastian/version ^3.0 -> satisfiable by sebastian/version[3.0.0].
    - Conclusion: don't install sebastian/version 3.0.0
    - Installation request for phpunit/phpunit ^9.0 -> satisfiable by phpunit/phpunit[9.0.0, 9.0.1].
```

but I believe there might be some package in my way. Thanks for help :)

**Comment by malohtie** [comment]
i solved this by upgrading phpUnit to ^9.0
